server.js

```diff
@@ -1,15 +1,22 @@
 const express = require('express');
 const app = express();
-const port = process.env.PUBLIC_PORT || 5000;
+const port =5000;
+const mongoose = require('mongoose');
 
-app.get("/ping",(req,res)=>{
-  res.send("pong")
+const uri= 'mongodb+srv://anushka_poonia:anushkapoonia3777@cluster0.rniykvz.mongodb.net/?retryWrites=true&w=majority'
+
+mongoose.connect(uri)
+.then(()=>{
+  app.get("/",(req,res)=>{
+    res.json({connection:"Connected"})
+  })
 })
 
-if (require.main === module) {
+.catch(()=>{
```

Comment:
It's a good practice to handle errors in a more detailed way. Consider logging the error or sending it in the response. [medium]